src/mantarray_desktop_app/ok_comm.py

```diff
@@ -512,14 +511,11 @@ def __init__(
         suppress_setup_communication_to_main: bool = False,
         logging_level: int = logging.INFO,
     ):
-        # pylint-disable: duplicate-code # needed for the type definition of the board_queues
-        super().__init__(fatal_error_reporter, logging_level=logging_level)
-        self._board_queues = board_queues
-        self._board_connections: List[Union[None, okCFrontPanel]] = [None] * len(
-            self._board_queues
-        )
-        self._suppress_setup_communication_to_main = (
-            suppress_setup_communication_to_main
+        super().__init__(
```

Comment:
possible idea for the future to avoid a bunch of duplicate code warnings for `init`: in the parent, have init call a method `_initialize_class_attributes()`.  Then in the subclasses, all the specific things like `_is_first_managed_read` could be declared as class attributes and then set to their initial value in that `_initialize_class_attributes()` method

Reply:
if an instance assigns a value to a class attribute using `self._my_attribute`, then (my understanding) it becomes an object specific to that instance, so we wouldn't need to worry about it crossing over during test cases (i.e. it's not actually acting like a singleton class attribute)